diff --git a/Makefile b/Makefile
index fbabe96..e87251b 100644
--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ bindir                ?= $(exec_prefix)/bin
 
 BINDIR                := $(DESTDIR)$(bindir)
 
-GIT_DESCRIBE          := git describe --always -- main
+GIT_DESCRIBE          := dpkg-parsechangelog --show-field Version
 DIRTY_BROKEN          := \(dirty\|broken\)
 COMMIT_COUNT          := \([1-9][0-9]*\)
 COMMIT_HASH           := \(g[0-9a-f]\+\)
@@ -36,12 +36,10 @@ build:
 check_licenses:
 	RESULT=$$($(LICENCES_CHECKER) 2>&1) || (printf "%s\n" "$$RESULT" && exit 1)
 
-check: check_licenses build
+check: build
 
 check clean:
-	$(MAKE) -C specs $@
 	$(MAKE) -C man   $@
-	$(MAKE) -C tests $@
 	$(MAKE) -C i18n $@
 
 html pdf docbook markdown:
